### ISSUE TITLE
what is the mask

### ISSUE THREAD

**Issue by WestBbbbrook** [piergiaj/super-events-cvpr18#9]
I didn't find the mask in the dataset，can you tell me what is it？
```
 # forward
    if not baseline:
        outputs = model([inputs, torch.sum(mask, 1)])
    else:
        outputs = model(inputs)
```

**Comment by kkahatapitiya** [comment]
Look at the collate function in dataset file. When batch_size>1, it pads temporally small input features in a batch to a common temporal size (max). Mask is 1 where for actual duration of the padded input feature, and 0 otherwise.
